src/converters/toZigbee.js

```diff
@@ -1077,9 +1077,20 @@ const converters = {
 
             const zclData = {transtime: utils.getTransition(entity, key, meta).time};
 
-            if (newColor.isRGB() || newColor.isXY()) {
-                // Convert RGB to XY color mode because Zigbee doesn't support RGB (only x/y and hue/saturation)
-                const xy = newColor.isRGB() ? newColor.rgb.gammaCorrected().toXY().rounded(4) : newColor.xy;
+            const supportsHueAndSaturation = utils.getMetaValue(entity, meta.mapped, 'supportsHueAndSaturation', 'allEqual', true);
+            const supportsEnhancedHue = utils.getMetaValue(entity, meta.mapped, 'supportsEnhancedHue', 'allEqual', true);
+
+            if (newColor.isRGB() || newColor.isXY() || (!supportsHueAndSaturation && newColor.isHSV())) {
+                // Convert RGB/HSV to XY color mode because Zigbee doesn't support RGB and the light does not support hue/saturation
+                let xy;
+                if (newColor.isRGB()) {
+                    xy = newColor.rgb.gammaCorrected().toXY().rounded(4);
+                } else if (newColor.isHSV()) {
+                    meta.logger.warn(`This device does not support Hue/Saturation, falling back to X/Y!`);
```

Comment:
Logging a warning on every color set in this case will be annoying (maybe logs as debug). But I'm wondering if we should do any conversion at all and just throw an error instead. I have not seen users asking for this feature yet.

Reply:
I was also leaning on just erroring, I will change this after work.

Reply:
Done:
```
Zigbee2MQTT:debug 2023-05-30 20:14:02: Received MQTT message on 'zigbee2mqtt/0x000d6ffffe197fe9/set' with data '{"color": {"hue": 128, "saturation": 100}}'
Zigbee2MQTT:debug 2023-05-30 20:14:02: Publishing 'set' 'color' to '0x000d6ffffe197fe9'
Zigbee2MQTT:error 2023-05-30 20:14:02: Publish 'set' 'color' to '0x000d6ffffe197fe9' failed: 'Error: This light does not support Hue/Saturation, please use X/Y instead.'
Zigbee2MQTT:debug 2023-05-30 20:14:03: Error: This light does not support Hue/Saturation, please use X/Y instead.
    at Object.convertSet (/Users/sjorge/Documents/repos/zigbee-herdsman-converters/src/converters/toZigbee.js:1085:23)
    at Object.convertSet (/Users/sjorge/Documents/repos/zigbee-herdsman-converters/src/converters/toZigbee.js:787:61)
    at Publish.onMQTTMessage (/Users/sjorge/Documents/repos/zigbee2mqtt/lib/extension/publish.ts:247:52)
    at EventEmitter.emit (node:events:523:35)
    at EventBus.emitMQTTMessage (/Users/sjorge/Documents/repos/zigbee2mqtt/lib/eventBus.ts:109:22)
    at MQTT.onMessage (/Users/sjorge/Documents/repos/zigbee2mqtt/lib/mqtt.ts:140:27)
    at MqttClient.emit (node:events:511:28)
    at MqttClient._handlePublish (/Users/sjorge/Documents/repos/zigbee2mqtt/node_modules/mqtt/lib/client.js:1547:12)
    at MqttClient._handlePacket (/Users/sjorge/Documents/repos/zigbee2mqtt/node_modules/mqtt/lib/client.js:535:12)
    at work (/Users/sjorge/Documents/repos/zigbee2mqtt/node_modules/mqtt/lib/client.js:438:12)
Zigbee2MQTT:info  2023-05-30 20:14:03: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'color' to '0x000d6ffffe197fe9' failed: 'Error: This light does not support Hue/Saturation, please use X/Y instead.'","meta":{"friendly_name":"0x000d6ffffe197fe9"},"type":"zigbee_publish_error"}'
```